examples/Console/TestMetrics.cs

```diff
@@ -66,11 +66,11 @@ internal static object Run(MetricsOptions options)
                 AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
 
                 providerBuilder
-                    .AddOtlpExporter(o =>
+                    .AddOtlpExporter((exporterOptions, metricReaderOptions) =>
```

Comment:
Maybe we could add some use of `exporterOptions` in this example, otherwise the proper code would replace this variable name with an underscore?

Reply:
Added an option to choose gRPC or HTTP